### ISSUE TITLE
kube-batch does not support nodeaffinity

### ISSUE THREAD

**Issue by wukong1992** [kubernetes-retired/kube-batch#516]
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>
> /kind bug

**What happened**:
pod always pending
**What you expected to happen**:
scheduled;status:running
**How to reproduce it (as minimally and precisely as possible)**:

```yaml
apiVersion: batch/v1
kind: Job
metadata:
  name: job1
spec:
  template:
    metadata:
      annotations:
        scheduling.k8s.io/group-name: g1
    spec:
      containers:
      - image: busybox
        imagePullPolicy: IfNotPresent
        name: busybox
        resources:
          requests:
            cpu: "1"
      restartPolicy: Never
      schedulerName: kube-batch
      affinity:
       nodeAffinity:
        preferredDuringSchedulingIgnoredDuringExecution:
        - weight: 1
          preference:
             matchExpressions:
              - key: cluster
                operator: In
                values:
                - 'test1'
        - weight: 1
          preference:
             matchExpressions:
              - key: cluster
                operator: In
                values:
                - 'test2'
---
apiVersion: scheduling.incubator.k8s.io/v1alpha1
kind: PodGroup
metadata:
  name: g1
spec:
  minMember: 3
```

**Anything else we need to know?**:
kube-batch version:v0.2
**Environment**:
- Kubernetes version (use `kubectl version`):1.10
- Cloud provider or hardware configuration:private
- OS (e.g. from /etc/os-release):centos
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by k82cn** [comment]
We support node affinity predicates in 0.3; node affinity prioritize is on going :)

**Comment by k82cn** [comment]
fixed by #587
